docs/adr/0002-videos-languages.rst

```diff
@@ -0,0 +1,64 @@
+ADR 0002 - Videos languages
+===========================
+
+Status
+------
+
+Proposed
+
+
+Context
+-------
+
+We want to think Marsha as accessible from the beginning. At least from the point of view of the videos, which are the main content available.
+
+We can think about a video as a "set of related videos with their accessibility counterparts".
+
+Counterparts
+~~~~~~~~~~~~
+
+Audio
++++++
+
+The same video can be proposed with different audio tracks. Which often result in many videos. That's our case, we don't manage audio tracks separately.
+
+Subtitles
++++++++++
+
+In addition to audio tracks, many subtitles tracks can be available.
+
+Sign language
++++++++++++++
+
+Some people with disabilities could want a video with the sign language transcript. For this it can be a video incorporated in the original one, or an other video displayed on the site.
+
+As sign languages are not the same for every spoken language, it can exist many sign languages videos for a single video.
+
+Summary
+~~~~~~~
+
+What we call a "video" can in fact have many videos, each one with a different audio-track. Each of these video can have different subtitles, and different sign-languages videos.
+
+
+Decision
+--------
+
+We decided to take all this elements into account right from the beginning.
+
+We'll use `django-parler <https://github.com/django-parler/django-parler>`_ to manage translations.
+
+So we have a main Django model named ``Video``, from an author. As a video can be in many languages (audio tracks), we use ``django-parler`` for this: the title and the video url are managed this way.
+
+For each of these videos, we can have subtitles (even if in the main case, subtitles will only be available for one video, because each video could have a different length, text, etc).
+
+So we have a new Django model named ``SubtitlesTrack`` (``Track`` suffix because a model should be singular, but it's not really *a* subtitle. ). It has a ``ForeignKey`` to the ``Video`` related instance, named ``video``, and a field named ``video_language`` to know for which language make these subtitles available.
```

Comment:
I think `SubtitleTrack` is fine and looks less like a failure to stick to singular ^^